server/pom.xml

```diff
@@ -193,6 +193,7 @@
         <cassandra-unit.version>2.1.9.2</cassandra-unit.version>
         <assertj-1.version>1.7.1</assertj-1.version>
         <assertj-3.version>3.2.0</assertj-3.version>
+        <throwing-lambdas.version>0.5.0</throwing-lambdas.version>
```

Comment:
usually we use dependencyManagement section for declaring versions, not properties